app/lib/realm.js

```diff
@@ -1,15 +1,39 @@
 import Realm from 'realm';
 
-// import { AsyncStorage } from 'react-native';
-// Realm.clearTestState();
-// AsyncStorage.clear();
+const email = {
+	name: 'email',
+	primaryKey: 'address',
+	properties: {
+		// _id: { type: 'int', optional: true },
+		address: { type: 'string', optional: true },
+		verified: { type: 'bool', optional: true }
+	}
+};
+
+const userSchema = {
+	name: 'user',
+	properties: {
+		id: { type: 'string', optional: true },
+		token: { type: 'string', optional: true },
+		username: { type: 'string', optional: true },
+		name: { type: 'string', optional: true },
+		language: { type: 'string', optional: true },
+		status: { type: 'string', optional: true },
+		customFields: { type: 'string', optional: true },
+		email: { type: 'list', objectType: 'email', default: [] },
+		roles: { type: 'string[]', optional: true }
+	}
+};
 
 const serversSchema = {
 	name: 'servers',
 	primaryKey: 'id',
 	properties: {
 		id: 'string',
 		name: { type: 'string', optional: true },
+		currentServer: { type: 'bool', optional: true },
+		userToken: { type: 'string', optional: true },
```

Comment:
you already declared `token` field in `user` schema on line 17 same file. So you can access `servers.userToken` using `servers.user.token. I think no need in duplicated token field value in two places.